releases/handbook.md

```diff
@@ -266,11 +266,50 @@ those can start to be documented.
+
 **Success Criteria:** Documentation for this release completed with minimum following pages updated and a [new version
 in the website is cut](https://github.com/kubeflow/kubeflow/blob/master/docs_dev/releasing.md#version-the-website).
 - [Installing Kubeflow](https://www.kubeflow.org/docs/started/installing-kubeflow/)
 - [Release Page](https://www.kubeflow.org/docs/releases/)
 - [Distributions](https://www.kubeflow.org/docs/distributions/) and related pages underneath
+- [Manifest Readme](https://github.com/kubeflow/manifests#readme)
+- [Manifest Releases](https://github.com/kubeflow/manifests/releases)
+- [Manifest Components](https://github.com/kubeflow/manifests#kubeflow-components-versions) 
+
+Change logs per Working Group
+
+- [Notebooks](https://github.com/kubeflow/kubeflow/releases)  
+- [Training Operator](https://github.com/kubeflow/training-operator/blob/master/CHANGELOG.md)  
+- [Katib](https://github.com/kubeflow/katib/blob/master/CHANGELOG.md)  
+- [Pipelines](https://github.com/kubeflow/pipelines/blob/master/CHANGELOG.md)  
+- [KServe](https://github.com/kserve/kserve/releases/)
+
+Working Group updates these pages (and the pages underneath) for the features changed in current release
+
+- [Notebooks](https://www.kubeflow.org/docs/components/notebooks/)
+- [Training Operator](https://www.kubeflow.org/docs/components/training/)
+- [Katib](https://www.kubeflow.org/docs/components/katib/)
+- [Pipelines](https://www.kubeflow.org/docs/components/pipelines/)
+- [KServe](https://kserve.github.io/website/)
+
+Release Team and Product Management updates
+
+- [Release Handbook](https://github.com/kubeflow/community/blob/master/releases/handbook.md)
+- [Kubeflow blog post](https://blog.kubeflow.org/)
+
+**Best practices for general documentation updates, documentation refactoring and distribution documentation.**
+
+*Non-release documentation updates*
+
+The Community has found that general or non-release documentation updates are often committed in the days prior to a release.  These non-release updates are often needed, but they add additional reviews and approvals during a very busy time.   As a best practice, documentation committers should try to have PRs ready for non-release updates prior to the Release feature freeze.
+
+*Long term re-factoring and distribution documentation*
+
+In the long term, the Community may want to re-factor the documentation, which might include separating the user functionality from the installation and administration documentation.   Additionally, each individual distribution documentation should have moved, or have a plan to move, their documentation from Kubeflow.org.    
```

Comment:
Should we also expose that as a best practice such long term re-factoring efforts, that might be huge as well, should not happen prior to the release? 

Right now we are cutting the release branches of the website before a release. So we can't have such big efforts getting merged in the last minute.